guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -427,7 +427,7 @@ Example of usage:
 </page>
 {%endhighlight xml%}
 
-### &lt;update&gt; {#fedg_layout_xml-instruc_ex_upd}
```

Comment:
I would not eliminate the "<" and the ">" in this title, it references a node inside layout.